Libplanet/Blocks/Block.cs

```diff
@@ -178,6 +158,9 @@ private Block(RawBlock rb)
                     rb.Header.Timestamp,
                     BlockHeader.TimestampFormat,
                     CultureInfo.InvariantCulture).ToUniversalTime(),
+#pragma warning disable MEN002 // Line is too long
+                rb.Header.TxHash.Any() ? new HashDigest<SHA256>(rb.Header.TxHash) : (HashDigest<SHA256>?)null,
+#pragma warning restore MEN002 // Line is too long
```

Comment:
Can't we expand this expression into multiple lines?

```suggestion
                rb.Header.TxHash.Any()
                    ? new HashDigest<SHA256>(rb.Header.TxHash)
                    : (HashDigest<SHA256>?)null,
```

Reply:
If then, we'll face [SA1118](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1118.md). it is okay to change it to ignore SA1118, but I chose to keep the existing code style.

Reply:
Okay. It's still confusing that the following argument has multiple lines though… 🤔:

```
                rb.Transactions
                    .Select(tx => Transaction<T>.Deserialize(tx.ToArray()))
                    .ToList(),
```